### ISSUE TITLE
no-unknown-classes silently reports every base utility as unknown when components.json tailwind.css is a partial that does not import tailwindcss

### ISSUE THREAD

**Issue by kang8** [shadcn-ui/lint#13]
### What happens

If `components.json` points `tailwind.css` at a stylesheet that does not itself
`@import "tailwindcss"` — a token-only partial, the normal shape for a component
package in a monorepo — the theme builds fine but holds no base utilities, so
`no-unknown-classes` reports every stock class as unknown. Silently: nothing on
stderr, and the message blames the call site (`"text-sm" is not a class this
project's Tailwind knows`). In our repo that was 198 errors across one package.

Related to #9, different cause: there the `@import` fails to resolve, which does
warn and fall back to the bundled grammar. Here nothing fails.

### Reproduction

```css
/* src/theme.css — the tailwind.css entry: tokens only */
@theme inline { --color-brand: #123456; }
```

```css
/* src/app.css — the real entry */
@import 'tailwindcss';
@source './';
@import './theme.css';
```

```tsx
// src/page.tsx
export function Page() {
  return <div className="flex p-4 text-sm bg-brand">Hello</div>
}
```

`eslint src/page.tsx` over the same tree, changing only `tailwind.css`:

| `tailwind.css` | result |
| --- | --- |
| `src/theme.css` | `p-4` and `text-sm` unknown — exit 1, no warning |
| `src/app.css` | clean |
| *`components.json` deleted* | clean |

Deleting the configuration fixes the rule, because `discoverThemeFile` then
finds `app.css` on its own.

### Cause

`discoverThemeFile` requires a candidate to import Tailwind:

```js
const read = themeAt(file);
if (!read.tailwind) continue;
```

`themeFileFor` never applies that to the configured path — `if (isFile(project.cssFile)) return project.cssFile`.
Same fact: hard requirement on one path, unchecked on the other.

### Suggested fix

Consult `themeAt(project.cssFile).tailwind` there too and warn when it is false;
the neighbouring missing-file `warnOnce` already uses the wording ("No
stylesheet importing Tailwind was found"). Whether to also fall back to
`discoverThemeFile` is a judgement call — it changes which file `no-raw-colors`
names in "declare `--color-<name>` in X", and for a library package the partial
is still where a token belongs. The warning alone would make this diagnosable.

`@shadcn/lint` 0.1.0, eslint 10, tailwindcss 4.3.3.

**Comment by leandromatos** [comment]
Reproduced with a component package whose stylesheet leaves `@import "tailwindcss"` to the consumer on purpose, so the preflight is never imported twice. Pointing `tailwind.css` at the one sheet in the package that does import Tailwind (the catalogue's entry) is the workaround; validating the configured path the way `discoverThemeFile` validates a candidate, as suggested, would have said so up front instead of reporting `px-3` as unknown.
